api/v1alpha1/cloud_storage_types.go

```diff
@@ -25,18 +25,19 @@ const (
 )
 
 type CloudStorageSpec struct {
-	// Name is the name requested for the bucket (aws, gcp) or container (azure)
+	// `name` is the name requested for the bucket (aws, gcp) or container (azure)
```

Comment:
@kaovilai I am wondering on why do we have strings in backticks ?

Reply:
lol.. no particular reason.

Reply:
oh ok, lets get rid of the backticks then.